test/presubmit-tests-with-pipeline-deployment.sh

```diff
@@ -126,6 +126,12 @@ ${KUBEFLOW_SRC}/scripts/kfctl.sh init ${KFAPP} --platform ${PLATFORM} --project
 
 cd ${KFAPP}
 ${KUBEFLOW_SRC}/scripts/kfctl.sh generate platform
+
+## Add one gpu node for covering gpu sample
```

Comment:
can we add a parameter and run this conditionally? for integration test we dont need to create gpu cluster, and it improve the test reliability.

https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubeflow/kubeflow-presubmits.yaml#L230

Reply:
the test covers gpu e2e and cluster setup. Previously, the test was flaky because of GCE stockout which is irrelevant to GPU test. I'd prefer to add the test back until we can prove it is the cause of test flakiness.